signature-envelope-jws.md

```diff
@@ -54,26 +54,51 @@ Example of Notary v2 payload
 The JWS envelope for Notary v2 uses following headers
 
 - Registered headers - `alg`, `cty`, and `crit`
-- [Public headers](https://datatracker.ietf.org/doc/html/rfc7515#section-4.2) with collision resistant names - `io.cncf.notary.signingTime`, `io.cncf.notary.expiry`
+- [Public headers](https://datatracker.ietf.org/doc/html/rfc7515#section-4.2) with collision resistant names.
+  - `io.cncf.notary.signingScheme`
+  - `io.cncf.notary.signingTime`
+  - `io.cncf.notary.authenticSigningTime`
+  - `io.cncf.notary.expiry`
 
-Example
+Example with Signing Scheme `notary.default.x509`
 
 ```jsonc
 {
     "alg": "PS384",
     "cty": "application/vnd.cncf.notary.payload.v1+json",
+    "io.cncf.notary.signingScheme": "notary.default.x509",
     "io.cncf.notary.signingTime": "2022-04-06T07:01:20Z",
     "io.cncf.notary.expiry": "2022-10-06T07:01:20Z",
-    "crit":["io.cncf.notary.expiry"]
+    "crit":[
+      "io.cncf.notary.signingScheme",
+      "io.cncf.notary.expiry"
```

Comment:
For expiry, does this need to be in the `crit` section. See: https://github.com/notaryproject/notaryproject/issues/141